jha/src/views/enrolment/SpouseInfoPage.vue

```diff
@@ -29,53 +30,59 @@
           <Input label='First name:'
             id='first-name'
             className='mt-3'
-            maxlength="18"
+            maxlength="30"
             v-model='spouseFirstName'
+            @blur="handleBlurField($v.spouseFirstName)"
             :inputStyle='mediumStyles' />
           <div class="text-danger"
             v-if="$v.spouseFirstName.$dirty && !$v.spouseFirstName.required"
             aria-live="assertive">First name is required.</div>
           <div class="text-danger"
             v-if="$v.spouseFirstName.$dirty && $v.spouseFirstName.required && !$v.spouseFirstName.nameValidator"
-            aria-live="assertive">First name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.</div>
+            aria-live="assertive">First name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.<br/>First name must be a letter.</div>
           <Input label='Middle name (optional):'
             id='middle-name'
             className='mt-3'
-            maxlength="18"
+            maxlength="30"
             v-model='spouseMiddleName'
+            @blur="handleBlurField($v.spouseMiddleName)"
             :inputStyle='mediumStyles' />
           <div class="text-danger"
             v-if="$v.spouseMiddleName.$dirty && !$v.spouseMiddleName.nameValidator"
-            aria-live="assertive">Middle name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.</div>
+            aria-live="assertive">Middle name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.<br/>First name must be a letter.</div>
           <Input label='Last name:'
             id='last-name'
             className='mt-3'
-            maxlength="18"
+            maxlength="30"
             v-model='spouseLastName'
+            @blur="handleBlurField($v.spouseLastName)"
             :inputStyle='mediumStyles' />
           <div class="text-danger"
             v-if="$v.spouseLastName.$dirty && !$v.spouseLastName.required"
             aria-live="assertive">Last name is required.</div>
           <div class="text-danger"
             v-if="$v.spouseLastName.$dirty && $v.spouseLastName.required && !$v.spouseLastName.nameValidator"
-            aria-live="assertive">Last name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.</div>
+            aria-live="assertive">Last name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.<br/>First name must be a letter.</div>
```

Comment:
The extra wrong error message should say "Last name" instead of "First name" here.